docs/snippets/common/storybook-preview-parameters-old-format.js.mdx

```diff
@@ -1,5 +1,7 @@
 ```js
 // .storybook/preview.js
 
-export const parameter = { passArgsFirst : false }.
+export default {
+  parameters: { passArgsFirst : false },
```

Comment:
Seems there's an extra space here as well, should be `parameters: { passArgsFirst: false },`